app/views/PremiumFeaturesView.coffee

```diff
@@ -20,9 +21,9 @@ module.exports = class PremiumFeaturesView extends RootView
     super(options)
     
   afterInsert: () ->
-    super()
-    if utils.getQueryVariable('pop')?
+    if utils.getQueryVariable('pop')? and not storage.load('sub-modal-continue')
       @openSubscriptionModal()
+    super()
```

Comment:
Why move `super()` to the bottom? Standard behavior elsewhere seems to be to keep it at the top of `afterInsert`

Reply:
super() calls the CocoView afterInsert, which opens a subscribe modal and removes this storage key.  Then we open another subscribe modal if pop query variable is also set.

Is an explanation comment here good enough?